### ISSUE TITLE
Homematic (Hörmann) Garage Door Opener not working via HAP —> Homekit?

### ISSUE THREAD

**Issue by schoe-to** [thkl/hap-homematic#662]
I added it, trying both with and without the separat option for a ventilation position but it is not added to the active device. Anybody can help me or has experience with this?

![IMG_0492](https://github.com/thkl/hap-homematic/assets/151659531/8dc2fcc5-5b45-49e7-8a85-966d3ea2ad2c)

**Comment by schoe-to** [comment]
Does it work for you with a HmIP-MOD-HO? @gerd2005

**Comment by gerd2005** [comment]
Yes, no problems. What about the Ports in the "Instanzen"?

![image](https://github.com/thkl/hap-homematic/assets/72799068/332d4bab-87dd-43c7-868c-3ec0ef5b89e0)
![image](https://github.com/thkl/hap-homematic/assets/72799068/5e0653e2-33ef-4cbe-a6ad-806ae5aca04e)

**Comment by schoe-to** [comment]
Strange. What do you mean by „Ports in the „Instanzen““? I have added all ports as needed to the Firewall exceptions and I was able to add e.g. a light to the same instance „Außenbereich“ where I would like to add the garage door as well. I was even able to add the „Hörmann“ light (Antriebslicht) that is managed via the HmIP-MOD-HO as well, but it is not possible to add the more important Garage Door Service itself :-/

**Comment by gerd2005** [comment]
Sorry, dann habe ich leider auch keine weiteren Ideen mehr...
Der MOD_HO hat ja auch nur zwei Kanäle, oder? Einmal das Tor, und einmal das Licht. Wenn der noch mehr hätte, könnte es ja auch sein, dass du den falschen Kanal hinzugefügt hast. Das denke ich aber nicht

**Comment by schoe-to** [comment]
Eine normale CCU3

**Comment by gerd2005** [comment]
Ok, dann weiss ich auch nicht. Aber im  #661 wird das Problem ja auch schon behandelt. Vielleicht wendet ihr euch direkt mal an thkl.

**Comment by schoe-to** [comment]
Ja, hab's auch gerade entdeckt. Hoffe er kann helfen. Wäre stark, wenn das funktioniert.
